tests/test_win_prob.py

```diff
@@ -0,0 +1,49 @@
+"""Tests Win Probability module."""
+
+import pytest
+
+from awpy import Demo
+from awpy.stats import win_probability
+
+class TestWinProbability:
+    """Tests the win probability calculations.
+    
+    https://www.hltv.org/matches/2369248/natus-vincere-vs-virtuspro-pgl-cs2-major-copenhagen-2024-europe-rmr-closed-qualifier-a
+    """
+
+    @classmethod
+    def setup_class(cls):
```

Comment:
We probably want to use something like `@pytest.fixture(scope="class")` decorators as shown in `test_parsers.py`

Reply:
We could use an existing demo, as well

Reply:
implemented pytest fixtures and now use existing demo